src/utils/isNumbery.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Check whether the value is a number or not
```

Comment:
We're checking if it's a number or a number-like string: i.e. when coerced to a number it `==` itself.